test/e2e_kubeadm/proxy_addon_test.go

```diff
@@ -75,6 +77,22 @@ var _ = Describe("proxy addon", func() {
 			cm := GetConfigMap(f.ClientSet, kubeSystemNamespace, kubeProxyConfigMap)
 
 			gomega.Expect(cm.Data).To(gomega.HaveKey(kubeProxyConfigMapKey))
+
+			// Check KubeProxyConfigMap19
+			// gets the ClusterConfiguration from the kubeadm kubeadm-config ConfigMap as a untyped map
+			m := getClusterConfiguration(f.ClientSet)
+
+			// Extract the kubernetesVersion
+			gomega.Expect(m).To(gomega.HaveKey("kubernetesVersion"))
+			k8sVersionString := m["kubernetesVersion"].(string)
+			k8sVersion, err := version.ParseSemantic(k8sVersionString)
+			if err != nil {
+				framework.Failf("error reading kubernetesVersion from %s ConfigMap: %v", kubeadmConfigName, err)
+			}
+
+			if k8sVersion.Major() >= 1 && k8sVersion.Minor() >= 9 {
+				gomega.Expect(cm.Data).To(gomega.HaveKey(kubeProxyConfigMapKey19))
+			}
```

Comment:
@lubinsz 
please explain the motivation for adding this test?

/cc @fabriziopandini

Reply:
should this be called `kubeProxyConfigMapKeyKubeconfig`?

Reply:
> @lubinsz
> please explain the motivation for adding this test?
> 
> /cc @fabriziopandini

KubeProxyConfigMap19 is created by kubeadm, and used for Kubernetes 1.9 and above.
It is defined in kubeadm/app/phases/addons/proxy/manifests.go.
And this configmap is installed in cmd/kubeadm/app/cmd/phases/init/addons.go
I think we can check it in kubeadm related test cases.

Reply:
> should this be called `kubeProxyConfigMapKeyKubeconfig`?

OK.

Reply:
the name of the constants in kubeadm/app/phases/addons/proxy/manifests.go is not important and can be changed. better the constant in this PR without the "19".

your test is adding the verification for the existence of the kubeconfig.conf key in the kube-proxy ConfigMap:
https://github.com/kubernetes/kubernetes/blob/5b47a92ad8668612d912ced2f0e6c658b9325727/cmd/kubeadm/app/phases/addons/proxy/manifests.go#L30

and the scope of the PR is a bit misleading.
inspecting the contents of the config map is OK, but this should not be bound to a version check as 1.9 is long out of support.

Reply:
@neolit123 
Done.